### PR TITLE
Ramneet

### PR DESCRIPTION
**Pull request by rgreywall**
just initial styling

### CODE REVIEW

File: comps/DefaultImage.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import { View, Image, Text, Button, Animated } from 'react-native';
+import styles from '../styles/DefaultImageStyles';
+
+export default function DefaultImage(props){
+  console.log("in default image")
+  const [op] = useState(new Animated.Value)
```
Comment (kathleenhung5) [inline comps/DefaultImage.js]: Hey Ramneet, you may want to put a starting value for animation. Right now you are animating from nothing to 1. So you should do useState(new Aniamted.Value(0)); if 0 is the value you want to start with.

Comment (rgreywall) [reply comps/DefaultImage.js]: thank you so much!

**Comment by j-ulienguyen** [comment]
I was fixing up the code so that we are able to see the component properly.
However, I don't think the animation is working as intended.

If someone can look over that please

**Comment by rgreywall** [comment]
thanks for helping Julie & Kathleen

**Pull request merged**